server/src/main/scala/passengersplits/polling/ManifestFilePolling.scala

```diff
@@ -175,11 +175,23 @@ object AtmosManifestFilePolling {
     def latestFile = latestFileName
   }
 
-
   def runSingleBatch(tickId: TickId, zipFilenamesSource: Source[String, NotUsed], unzipFileContent: UnzipFileContentFunc[String],
                      flightPassengerReporter: ActorRef, batchFileState: BatchFileState,
                      batchAtMost: FiniteDuration,
                      voyageManifestFilter: (VoyageManifest) => Boolean)(implicit actorSystem: ActorSystem, materializer: Materializer): Unit = {
+    Try {runSingleBatch(tickId, zipFilenamesSource, unzipFileContent, flightPassengerReporter, batchFileState, batchAtMost,
+      voyageManifestFilter)} match {
+      case Success(s) => log.info(s"tickId: $tickId runSingleBatch success with $s")
+      case Failure(f) => log.warn(s"tickId: $tickId runSingleBatch failure with $f", f)
+    }
+  }
+
+
+    def innerRunSingleBatch(tickId: TickId, zipFilenamesSource: Source[String, NotUsed], unzipFileContent: UnzipFileContentFunc[String],
```

Comment:
you defined this function but haven't called it - did you add everything to the commit that you meant to?